scripts/send_to_bot.py

```diff
@@ -18,11 +18,6 @@ def post_comment(pr_id, comment_body):
   if f.getcode() != 200:
     print(f.read())
 
-
-# Get the environment variables
-PR_NUMBER = os.environ.get('TRAVIS_PULL_REQUEST')
-BUILD_ID = os.environ.get('TRAVIS_BUILD_ID')
-
```

Comment:
I'm not sure that moving these to be inline is a good idea. The general idea here is to separate concerns. If in the future we wanted to change the way we got those values, or needed them in more places than one, you'd have a problem if they weren't separated.

Additionally, it's useful for reference to have them all in one place.